packages/story-editor/src/app/media/media3p/attribution.js

```diff
@@ -94,33 +94,41 @@ const getAriaLabel = (provider) =>
 
 export function UnsplashAttribution() {
   return (
-    <a href={unsplashUrl} target={'_blank'} rel={'noreferrer'}>
-      <AttributionPill>
+    <a
+      href={unsplashUrl}
+      target={'_blank'}
+      rel={'noreferrer'}
+      aria-label={getAriaLabel(MEDIA_PROVIDER.unsplash)}
```

Comment:
aria-labels are better on elements that can be interacted with, aXe will flag nested aria-labels on spans as issues. This gives context to the anchor and then tells screen readers that it can ignore the content nested since it's just decorative.